sonar-csharp-plugin/src/test/java/org/sonar/plugins/csharp/CSharpSonarRulesDefinitionTest.java

```diff
@@ -38,7 +38,7 @@ public void test() {
     Context context = new Context();
     assertThat(context.repositories()).isEmpty();
 
-    CSharpSonarRulesDefinition csharpRulesDefinition = new CSharpSonarRulesDefinition(SonarVersion.SQ_67_RUNTIME);
+    CSharpSonarRulesDefinition csharpRulesDefinition = new CSharpSonarRulesDefinition(SonarVersion.SQ_79_RUNTIME);
```

Comment:
I guess this should be `SQ_LTS_RUNTIME` ?

Reply:
I wasn't sure about it. The LTS version was used indeed but the test name lacks to provide the context. It seems to be more about rule S100 than the sonar qube version.

Not sure what version we should use here.